agents/hwp_rotation/src/set_direction

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/expect
```

Comment:
I'm interested in this script, since it's different than the rest of the bash scripts that just call telnet. Can you describe a bit what's going on here?

Reply:
This was made during development but no longer serves a purpose. It has been remove